catkit/hardware/boston/BostonDmController.py

```diff
@@ -32,6 +31,8 @@ def initialize(self, serial_number, command_length, dac_bit_width):
         # Create class attributes for storing individual DM commands.
         self.dm1_command = None
         self.dm2_command = None
```

Comment:
These should be replaced by
```python
@property
def dm1_command(self):
  return self.dm1_command_object.to_command()

@property
def dm2_command(self):
  return self.dm2_command_object.to_command()
```

Once ``_open()`` calls ``apply_shape_to_both()`` instead of ``send_data()`` and there's a way for the controller to know the number of actuators (could prob do something with the ``command_length`` but that seemed yucky).